Plugin/CleverTapUnity/CleverTapUnitySDK/CleverTap.cs

```diff
@@ -0,0 +1,364 @@
+﻿using CleverTap.Utilities;
+using CleverTapUnitySDK.Common;
+using System.Collections.Generic;
+
+namespace CleverTapUnitySDK {
```

Comment:
Can we change namespace to be `CleverTap` only ?

Reply:
Changed